### ISSUE TITLE
survey.routines.inquire doesn't return bool

### ISSUE THREAD

**Issue by jmurrayufo** [Exahilosys/survey#18]
Version: `survey==4.5.1`
Python Version: `Python 3.11.4`

Sample Code:

```
import survey

x = survey.routines.inquire("Will you error?")

print(x)
```

In `4.5.0` this returned a Bool as expected per documentation, in `4.5.1` this returns a string.

**Comment by Exahilosys** [comment]
Hello, I do not seem to be able to reproduce this in `4.5.1`
![image](https://github.com/Exahilosys/survey/assets/22274370/cffbe289-8fe6-4c77-a93f-9153f2f8d409)

**Comment by Exahilosys** [comment]
Closing this with missing follow-up.

**Comment by jmurrayufo** [comment]
Sorry I missed the reply here. 

Which version of python are you running against?

Just trying this on a fresh docker instance to be sure. I main on python:3.11 but it seems to fail in python:3.10 as well.

```
docker run -it --rm python:3.10 bash
Unable to find image 'python:3.10' locally
3.10: Pulling from library/python
d52e4f012db1: Already exists 
7dd206bea61f: Already exists 
2320f9be4a9c: Already exists 
6e5565e0ba8d: Already exists 
d3797e13cc41: Already exists 
f8108bc94b5f: Pull complete 
3d35965ce4db: Pull complete 
4459f6d56468: Pull complete 
Digest: sha256:1b2e0805e24189fbba4e55b9bee89e3c25533cbe4fb71ae151f3e7ae0c9b86c5
Status: Downloaded newer image for python:3.10
root@dcb03e7d4652:/# pip install survey==4.5.1
Collecting survey==4.5.1
  Downloading survey-4.5.1-py3-none-any.whl (59 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 59.2/59.2 kB 3.1 MB/s eta 0:00:00
Installing collected packages: survey
Successfully installed survey-4.5.1
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

[notice] A new release of pip is available: 23.0.1 -> 23.2
[notice] To update, run: pip install --upgrade pip
root@dcb03e7d4652:/# python3
Python 3.10.12 (main, Jul  4 2023, 05:58:45) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> x = survey.routines.inquire("Will you error?")
KeyboardInterrupt
>>> import survey
>>> x = survey.routines.inquire("Will you error?")
? Will you error?Yes
>>> x
'y'
>>> 
```

**Comment by Exahilosys** [comment]
Hello, I was using 3.11.3. I have been researching the cause of this issue and I am a bit stumped, I cannot understand why pip is installing a seemingly older version of the library under the same version number for different python versions.

**Comment by Exahilosys** [comment]
Turns out I was reinstalling the library using a cached version of it and not the latest, should be fixed in [16765b6](https://github.com/Exahilosys/survey/commit/16765b6e4c359f66e830098000f297cfce2bf28c).

**Comment by jmurrayufo** [comment]
Fixed in `4.5.2` for me! Thanks!
